client/components/clipboard-button-input/index.jsx

```diff
@@ -51,13 +53,14 @@ export default React.createClass( {
 	},
 
 	render() {
-		const { value, className, disabled } = this.props;
+		const { value, className, disabled, hideHttp } = this.props;
 		const classes = classnames( 'clipboard-button-input', className );
 
 		return (
 			<span className={ classes }>
 				<FormTextInput
-					{ ...omit( this.props, 'className' ) }
+					{ ...omit( this.props, 'className', 'hideHttp' ) }
+					value={ hideHttp ? withoutHttp( value ) : value }
```

Comment:
I'm having second thoughts about this tweak to the ClipboardButtonInput component - it works for this case but feels like more of a shoe-horned solution than a proper, reasonable one.

I'd love to hear opinions on it but I'm putting the PR on hold for the time being.

There was discussion of changing how the URL bar displays - some of the ideas being considered would remove the need to hide the scheme in the ClipboardInput, instead showing the URL sans-scheme in plain text. We'd be making the change at implementation level, rather making more general changes to a component needlessly.

One such solution might look a little like this:
![clipboardhover](https://user-images.githubusercontent.com/4335450/28499634-27f21be2-6fb2-11e7-9d3b-1648afabc112.gif)